std_msgs/Header header
float64[] hit_scores
float64[] time_to_hit
int64[] pred_category
float64[] similarity
bool[] push_request
bool[] pull_request
int64 hit_index